docs/source/5-administrator.md

```diff
@@ -234,6 +237,14 @@ FlowMachine usage log:
 
 Note that the `request_id` field is identical across the five log entries, which lets you match the request across the multiple services.
 
+### Error and Debugging Logs
+
+By default, FlowMachine and FlowAPI write error logs to stdout. For more verbose logging, set the `LOG_LEVEL` environment variable to `debug` when starting the docker container.
+
+FlowMachine also supports writing debug and error logs to a file. To enable this, set the `WRITE_LOG_FILE` environment variable to `TRUE` when starting the FlowMachine container. Logs will be written to `$LOG_DIRECTORY/flowmachine-debug.log` (by default `LOG_DIRECTORY=/var/log/flowmachine`). If the directory specified does not exist, FlowMachine will attempt to create it.
```

Comment:
Just to be extra explicit, we should mention that this directory lives inside the docker container, e.g.: "Logs will be written to `$LOG_DIRECTORY/flowmachine-debug.log` inside the docker container (by default `LOG_DIRECTORY=/var/log/flowmachine`)."

Come to think of it, why do we need to allow configuring this directory at all if it's inside the container? Wouldn't it be simpler and more predictable to always write to the same directory? The user can always mount a custom host directory at `/var/log/flowmachine` if they also want to store the logs outside the container.

Reply:
Is there any situation in which we would ever want to run flowmachine outside of a container? If so, it might be necessary to change the log directory in that situation.

Reply:
Fine with mentioning explicitly that this is inside the container, but I think worth leaving it configurable because otherwise it makes it a pain should you wish to run it _outside_ a container. Not our presumed deployment scenario, but a totally reasonable thing to do (and indeed, we do it ourselves in the tests).